src/main.ml

```diff
@@ -1620,6 +1620,16 @@ let main () =
     "--tlkcmp-to", Myarg.String (fun s -> tlkcmp_dest := Some(s)), "X\t... into this one";
     "--tlkcmp-use-strings", Myarg.Set tlkcmp_strings, "\tmodifies --tlkcmp behavior";
 
+    "", Myarg.Unit (fun a -> a), "\nModder Options:\n" ;
+    "SETUP_TRA", Myarg.Unit (fun a -> a), "Can be set to NONE, WARN FAIL";
+    "AREA_VARIABLES", Myarg.Unit (fun a -> a), "Can be set to NONE, WARN FAIL";
+    "MISSING_EXTERN", Myarg.Unit (fun a -> a), "Can be set to NONE, WARN FAIL";
+    "MISSING_RESREF", Myarg.Unit (fun a -> a), "Can be set to NONE, WARN FAIL";
+    "ICT2_ACTIONS", Myarg.Unit (fun a -> a), "Can be set to NONE, WARN FAIL";
+    "MISSING_EVAL", Myarg.Unit (fun a -> a), "Can be set to NONE, WARN FAIL";
+    "OVERWRITING_FILE", Myarg.Unit (fun a -> a), "Can be set to NONE, WARN FAIL";
+    "FUN_ARGS", Myarg.Unit (fun a -> a), "Can be set to NONE, WARN FAIL";
```

Comment:
You missed one comma or "or" in all the option lists. And it would look better with a tab prepended like in other options.

Reply:
Sure added